### PR TITLE
[SYM-4574] Add validate_event

### PR DESCRIPTION
**Pull request by llam15**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?
There is no command currently to check if an event succeeded. 

In addition, refactors the jobs and commands so that a command can be run independently. Currently the `mkdir -p sym` command that is required for each command is only run in the job, so users need to create that directory manually if they want to use the commands directly instead of the job. 

Issue Number: SYM-4574

## What is the new behavior?
This PR ensures that commands can be used directly and that the jobs are just convenience wrappers.
This PR adds functionality to verify if a Sym Event (request/deescalate) succeeded 

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by llam15** [comment]
Current dependencies on/for this PR:
* main
  * **PR #7** <a href="https://app.graphite.dev/github/pr/symopsio/sym-orb/7" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/symopsio/sym-orb/7?utm_source=stack-comment).

File: src/commands/validate_event.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+description: >
+  Validates that an event succeeded by polling the Sym Events API until we receive an event status of success or error.
+  The command will fail if an error status is received.
+  The environment variable SYM_JWT must be set for this orb to work.
+  For instructions on how to generate a Sym Token, see: https://docs.symops.com/docs/using-bot-tokens
+parameters:
+  event_id:
+    type: string
+    description: |
+      The ID of the Run to de-escalate. One of event_id or request_slug must be defined.
```
Comment (Arianna2028) [inline src/commands/validate_event.yml:10]: ?
```suggestion
      The ID of the event to validate. One of event_id or request_slug must be defined.
```

File: src/jobs/validate_event.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  Validates that an event succeeded by polling the Sym Events API.
+
+docker:
+  - image: cimg/deploy:2023.02
+
+parameters:
+  event_id:
+    type: string
+    description: |
+      The ID of the Run to de-escalate. One of event_id or request_slug must be defined.
```
Comment (Arianna2028) [inline src/jobs/validate_event.yml:11]:
```suggestion
      The ID of the event to validate. One of event_id or request_slug must be defined.
```

**Review by Arianna2028 — APPROVED** [review]

File: src/commands/validate_event.yml (hunk 1 above)
Comment (llam15) [reply src/commands/validate_event.yml:10]: whoops good catch. Missed that description update

**Pull request merged**